spec/negotiation.html

```diff
@@ -332,7 +332,7 @@
         1. Let _value_ be ? Get(_opts_, _property_).
         1. If _value_ is not *undefined*, then
           1. Let _value_ be ? ToNumber(_value_).
-        1. If _value_ is *NaN* or less than _minimum_ or greater than _maximum_, throw a *RangeError* exception.
+          1. If _value_ is *NaN* or less than _minimum_ or greater than _maximum_, throw a *RangeError* exception.
```

Comment:
interesting, this algo has been like that for a long time, since 1st edition. good catch!